README.md

```diff
@@ -162,7 +176,8 @@ Run the unit tests with jasmine:
 - `0.3.0` Non-breaking update, aligns AES encryption with openssl.
 - `0.3.1` Non-breaking update, additional test coverage and small fixes.
 - `0.4.0` Non-breaking update, add local disk caching of configuration.
+- `0.5.0` Major update, uses the ini file format of the cloco-cli tool.
 
 # License
-Copyright (c) 2016 345 Systems LLP
+Copyright (c) 2016-2017 345 Systems LLP
```

Comment:
one of these days you're going to have to explain to me how these work, I've always wondered :-)